pom.xml

```diff
@@ -66,7 +66,6 @@
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <jenkins.version>2.452.3</jenkins.version>
-    <jenkinsRuleTimeout>0</jenkinsRuleTimeout>
     <skip.surefire.tests>${skipTests}</skip.surefire.tests>
```

Comment:
:raised_eyebrow: whatever this does, it would not normally be present in a plugin POM.